### DISCUSSION TITLE
NSPANEL choice Normal/Pro US/EU

### DISCUSSION THREAD

**Discussion by pbranly** [Blackymas/NSPanel_HA_Blueprint#507] [Q&A]
Hi,
I am interested with nspanel and have the choice between standard and pro version : Is the application compatoible with the pro version ?
There is also the EU version and the US version. I think that the software gives the possibility to choose the version but is the mechanical part compatible with European (French) box ?

Thank you by advance
Phil

**Comment by deejaybeam** [comment]
The NSPanel PRO is a total different device.
You could also not use your android - phone as a replacement for the NSPanel - NSPanel is just an android device.
So - not possible with this project.

**Reply by pbranly** [reply]
A ok thank you 
So choice is between the eu and us version 
Phil

**Comment by edwardtfn** [comment]
> I think that the software gives the possibility to choose the version

That's right. You just have to choose the right TFT file to update to your panel.

> but is the mechanical part compatible with European (French) box ?

I'm pretty sure the part going inside the wall is exactly same size regardless if you use the EU or US versions. I have both models and can't see a difference, although I've never look at those side by side.

**Reply by edwardtfn** [reply]
I've provided more details around this on #713
